### PR TITLE
dev: add black to pre commit

### PR DESCRIPTION
**Pull request by jandom**
Adding black to pre-commit-config

### CODE REVIEW

**Review by lilyminium — APPROVED** [review]
LGTM, thanks @jandom.

You have quite a few PRs approved and ready to merge now, if you want to handle them! We typically squash-merge to avoid lots of small commits with minor changes (not relevant to this PR but some others are a bit larger).

**Review by RMeli — CHANGES_REQUESTED** [review]
LGTM, thanks @jandom! It just need the commit of #286 (once squash-merged) in the `.git-blame-ignore-revs` and then it's good to go!

File: .pre-commit-config.yaml
Comment (jandom) [inline .pre-commit-config.yaml:14]: forgot this about curve-ball: isort and black can be in conflict without this

**Comment by jandom** [comment]
@RMeli this should be cleaned-up now, PTAL

File: .pre-commit-config.yaml
Comment (RMeli) [reply .pre-commit-config.yaml:14]: Good catch!

**Pull request merged**
